src/Data/Quote/QuoteValuesConfig.ts

```diff
@@ -0,0 +1,161 @@
+    'RFC_TOD_EXW',
+    'RFC_TOD_FAS',
+    'RFC_TOD_FCA',
+    'RFC_TOD_FH',
+    'RFC_TOD_FOB',
+    'RFC_TOD_UN',
+  ],
+  termsOfPayment: [
+    'PSA_TOP_10',
+    'PSA_TOP_20',
+    'PSA_TOP_50',
+    'PSA_TOP_60',
+    'PSI_TOP_00',
+    'PSI_TOP_10',
+    'PSI_TOP_20',
+    'PSI_TOP_30',
+    'PSI_TOP_40',
+    'RFC_TOP_NT60',
+  ],
+  type: [
+    'PSA_QUO_STY_GEN_OVR',
+    'PSA_QUO_STY_INF',
+    'PSA_QUO_STY_NEW',
+    'PSA_QUO_STY_SRV',
+    'PSA_QUO_STY_STD',
+    'PSA_QUO_STY_SVC',
+  ],
+})
+
+provideAttributeLocalizers(Quote, {
+  mainStatus: {
+    PSA_SAP_MAI_QUO_LST: 'Quote lost',
+    PSA_SAP_MAI_QUO_ORD_CER: 'Quote generation',
+    PSA_SAP_MAI_QUO_ORD_CRE: 'Offer won',
+    PSA_SAP_MAI_QUO_ORD_EPH_VH: 'Decision phase',
+    PSA_SAP_MAI_QUO_ORD_EPH: 'Decision phase',
+    PSA_SAP_MAI_QUO_ORD_PRP_ZUR: 'Quote deferred',
+  },
+  status: {
+    PSA_PRO_QUO_AS6: 'quote OK, project finish in 3-6 months',
+    PSA_PRO_QUO_CMV: 'quote deferred by customer',
+    PSA_PRO_QUO_COK: 'quote won/customer promise',
+    PSA_PRO_QUO_CPE: 'quote submitted, pending',
+    PSA_PRO_QUO_EN: 'Decision phase negative',
+    PSA_PRO_QUO_EP: 'Positive decision phase',
+    PSA_PRO_QUO_LST: 'quote lost',
+    PSA_PRO_QUO_ORC: 'order placement in progress by the customer',
+    PSA_PRO_QUO_PSI_0: 'In ERP entered',
+    PSA_PRO_QUO_PSI_1: 'In ERP scheduled/allocated',
+    PSA_PRO_QUO_PSI_2: 'In ERP with invoice',
+    PSA_PRO_QUO_PSI_3: 'In ERP partial delivery',
+    PSA_PRO_QUO_PSI_4: 'In ERP partial delivery/partial invoice',
+    PSA_PRO_QUO_PSI_5: 'In ERP partial delivery/full invoice',
+    PSA_PRO_QUO_PSI_6: 'In ERP delivery',
+    PSA_PRO_QUO_PSI_7: 'In ERP delivery/partial invoice',
+    PSA_PRO_QUO_PSI_8: 'In ERP canceled',
+    PSA_PRO_QUO_PSI_9: 'In ERP completed',
+    PSA_PRO_QUO_QT2: 'quote among Top 2, purchase in less than 3 months',
+    PSA_PRO_QUO_VER: 'discarded; copied to new version',
+    PSA_PRO_QUO_WON: 'quote won and qualified to order',
+    PSA_PRO_QUO_WRK: 'quote preparation',
+  },
+  termsOfDelivery: {
+    FOA: 'FOB Airport Named airport of departure',
+    FOR: 'Free on Rail Named departure point',
+    PSI_TOD_DAP: 'Delivered At Place',
+    PSI_TOD_DAT: 'Delivered At Terminal',
+    RFC_TOD_CFR: 'cost and freight',
+    RFC_TOD_CIF: 'cost, insurance & freight',
+    RFC_TOD_CIP: 'carriage and insurance paid to',
+    RFC_TOD_CPT: 'carriage paid to',
+    RFC_TOD_DAF: 'delivered at frontier',
+    RFC_TOD_DCP: '',
+    RFC_TOD_DDP: 'delivered duty paid',
+    RFC_TOD_DDU: 'delivered duty unpaid',
+    RFC_TOD_DEQ: 'delivered ex quay',
+    RFC_TOD_DES: 'delivered ex ship',
+    RFC_TOD_EXQ: '',
+    RFC_TOD_EXW: 'ex works',
+    RFC_TOD_FAS: 'free alongside ship',
+    RFC_TOD_FCA: 'free carrier',
+    RFC_TOD_FH: '',
+    RFC_TOD_FOB: 'free on board',
+    RFC_TOD_UN: '',
+  },
+  termsOfPayment: {
+    PSA_TOP_10: 'cash payable within 14 days',
+    PSA_TOP_20: 'cash',
+    PSA_TOP_50: 'by C.O.D.',
```

Comment:
`PSA_TOP_40` seems to be missing from the list. Could you check, if there are maybe other values missing as well?

Reply:
You misread: `PSI_TOP_40` line 150

Reply:
I can see this value on the following URL: https://edit.scrivito.com/d0a154d76edf2a7bd991fc658e700a1d~https://dave-new-enum-localization.scrivito-portal-app.pages.dev/quote-details-0092ff67875b69f8?quote_id=4E1FEAAC57E7DFC7E040007F010032F5&_scrivito_workspace_id=hfaa80b41e53c154&_scrivito_display_mode=diff

<img width="1624" alt="Screenshot 2024-03-18 at 09 39 32" src="https://github.com/Scrivito/scrivito-portal-app/assets/86275/66d8df99-e6b9-40c2-84c3-353ccf46c2d5">

Please investigate.

Reply:
The value `PSA_TOP_40` is clearly in use, e.g. on https://github.com/Scrivito/scrivito-portal-app/pull/329/files#diff-a0bc8e033a6d719f20539e57301bc559e7a181e8729ac9262efdd95795af5d15R21.

Reply:
Clarified. The value `PSA_TOP_40` is invalid. I fixed the data. REVIEW